tests/wpt/webgl/meta/conformance2/textures/image/tex-2d-r11f_g11f_b10f-rgb-float.html.ini

```diff
@@ -0,0 +1,2 @@
+[tex-2d-r11f_g11f_b10f-rgb-float.html]
+  expected: ERROR
```

Comment:
What JS error is triggering in all of these failing tests?

Reply:
They are crashes: 
`internal error: entered unreachable code: Unsupported formats R11fG11fB10f Float (thread WebGL thread, at components/canvas/webgl_thread.rs:2987)`